### ISSUE TITLE
[Accessibility] Date pickers are not accessible using shortcut keys (PageUp, PageDown, Ctrl+PageUp & Ctrl+PageDown

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [OfficeDev/office-ui-fabric-react#10908]
User Experience:

Users who rely on screen reader are getting affected as Date pickers should be accessible using  shortcut keys.
 
Repro Steps:
Step 1: Turn on narrator and Open latest fabric Ui URL in Edge browser.
Step 2: Navigate to Date picker page
Step 3: Navigate to any date picker control and verify whether shortcut keys are working.

Actual:
Date pickers are not accessible using  shortcut keys (PageUp, PageDown, Ctrl+PageUp & Ctrl+PageDown) .

Expected:
Date pickers should be accessible using  shortcut keys (PageUp, PageDown, Ctrl+PageUp & Ctrl+PageDown) .
i.e. 
 PageUp: To navigate to Next month
 PageDown: To navigate to Previous month
 Ctrl+PageUp: To navigate to Next year
Ctrl+PageDown To navigate to Previous year.
 
![image](https://user-images.githubusercontent.com/53210862/67101442-41799980-f1df-11e9-98e2-5cf940eceb3b.png)

**Comment by marygans** [comment]
Hi @SyedRehmanRazvi thank you for submitting this issue!

I was able to repro this issue. Im assuming the expectation on behavior should match the [aria example](http://w3c.github.io/aria-practices/examples/dialog-modal/datepicker-dialog.html). 

Will ensure this gets added, thanks!

**Comment by ecraig12345** [comment]
It's probably best if @lorejoh12 or another calendar team member works on this, rather than taking it on ourselves. They would also be better able to answer whether it's something they intend to support.

**Comment by lorejoh12** [comment]
It's a feature request, but not one that I think would be particularly difficult to add. I'll add a note to our backlog to see if/when we would be able to pick this up.

**Comment by lorejoh12** [comment]
For some reason the PR merging didn't auto close the issues... closing, this should be fixed now.
